FreeRTOS/Demo/RISC-V_Galois_P1/link.ld

```diff
@@ -74,14 +74,15 @@ OUTPUT_ARCH( "riscv" )
 
 _STACK_SIZE = DEFINED(_STACK_SIZE) ? _STACK_SIZE : 4K;

```

Comment:
These should be changed to reflect the values in FreeRTOSConfig.h 
if there is no good way to do "ifdef" in the linker script, we should probably make a specific .ld script just for testgen

Reply:
I don't think there is a way to ifdef in linker scripts. And I am not using these memory configuration in all the tests. Only for SSL on 64-bit. I do use an additional `.ld` script to define these macros. That is why I modified `LDFLAGS =` to `LDFLAGS +=`.